service/healthz/metrics.go

```diff
@@ -0,0 +1,29 @@
+package healthz
```

Comment:
How do we use the metrics than? Do we have dashboards where we check the status codes of the health check endpoints?